config/webpack/shared.js

```diff
@@ -0,0 +1,79 @@
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+var webpack = require('webpack')
+var path = require('path')
+var process = require('process')
+var glob = require('glob')
+var extname = require('path-complete-extname')
+var distDir = process.env.WEBPACK_DIST_DIR
+
+if(distDir === undefined) {
+  distDir = 'packs'
+}
+
+config = {
+  entry: glob.sync(path.join('app', 'javascript', 'packs', '*.js*')).reduce(
+    function(map, entry) {
+      var basename = path.basename(entry, extname(entry))
+      map[basename] = [
+        "jquery-ujs",
+        "babel-polyfill",
+        path.resolve(entry)
+      ]
+      return map
+    }, {}
+  ),
+
+  output: { filename: '[name].js', path: path.resolve('public', distDir) },
+
+  module: {
+    rules: [
+      {
+        test: /\.js(.erb)?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            [ 'latest', { 'es2015': { 'modules': false } } ]
```

Comment:
Doesn't look like you've removed the `.babelrc` which specifies different options than this. Probably need to axe it and move that stuff here? Or if this config respects the `.babelrc`, move those options there? Or explain what's goin' on here so I feel better.